packages/PlanLimitsUI/src/pages/Limits/map.tsx

```diff
@@ -121,12 +122,26 @@ export default function LimitsMap({
           'minimumFlowLimitBoundaries',
           'name'
         );
-        const flowRestrictionsLevel = findFeature(
+
+        const flowRestrictionsAmount = findFeature(
           result,
           'minimumFlowLimitBoundaries',
           'plan_minimum_flow_value'
         );
 
+        const flowRestrictionsUnit = findFeature(
+          result,
+          'minimumFlowLimitBoundaries',
+          'plan_minimum_flow_unit'
+        );
+
+        const flowRestrictionsLevel = flowRestrictionsAmount
+          ? waterQuantity(
+              Number(flowRestrictionsAmount),
+              flowRestrictionsUnit as string
+            )
+          : undefined;
```

Comment:
Was explicitly not trying to tidy this up yet, just getting it working with the formatting